examples/oidc_cli/src/main.rs

```diff
@@ -422,6 +454,79 @@ impl OidcCli {
         Ok(())
     }
 
+    async fn sliding_sync(&self) -> anyhow::Result<()> {
```

Comment:
Since this is an example, I believe the purpose of this method should be documented, even without going into details.